### DISCUSSION TITLE
Ajout d'un pet 5 étoile

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#2110] [Idées acceptées / déjà implémentées]
Je me disais qu'il n'y avait pas de phénix dans le jeu.

Alors du coup la suggestion c'est de l'ajouter

(j'ai pas d'idée pour d'autre mais ça pourrait être sympa vu qu'on est de + en + à avoir des 5 étoiles, puis ça ajouterais de la diversité un peu)

**Comment by Feiryn** [comment]
Faut attendre que l'emoji sorte officiellement sur discord mais je suis pour

**Reply by Bombafox** [reply]
Je l'ai enfin sur tel, y'a un monde.
🐦‍🔥

**Comment by Nargacaura** [comment]
Maintenant que l'emoji du phoenix est disponible sur Discord, je n'sais pas si on peut déjà le mettre dans "Idées acceptées". En tout cas, j'suis aussi pour l'ajout de ce familier.

**Reply by BastLast** [reply]
c'est prévu de l'ajouter en v5 en effet

**Reply by Bombafox** [reply]
👀
